wp-content/themes/core/assets/pcss/typography/_mixins.pcss

```diff
@@ -104,3 +104,31 @@
 	line-height: 1.6;
 	font-weight: 400;
 }
+
+/* -------------------------------------------------------------------------
+ * Inline Links
```

Comment:
@dpellenwood I couldn't decide where this belongs. It felt odd to set up `/pcss/actions/_mixins.pcss` with just one mixin that relates to typography. Let me know your thoughts on architecture approach.

Reply:
Hmm.... I do like the `actions` naming from SquareOne. Even if the partial is only a single mixin for now. It's stubbed out nicely for future additions.

Reply:
True, I can see us wanting to use mixins for buttons/inputs and apply those to third party plugins/blocks like Gravity Forms. It is nice to have it included for future additions. I'll add.